objectstorage/objectstorage.go

```diff
@@ -0,0 +1,123 @@
+package objectstorage
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/vocdoni/saas-backend/db"
+)
+
+type ObjectStorageConfig struct {
+	DB        *db.MongoStorage
+	ServerURL string
+}
+
+type ObjectStorageClient struct {
+	db        *db.MongoStorage
+	serverURL string
+}
+
+var isObjectNameRgx = regexp.MustCompile(`^([a-zA-Z0-9]+)\.(jpg|jpeg|png)`)
```

Comment:
why does the regex capture the extension of the object instead of just the objectID? i cannot find where the file extension is extracted and used with this regex.

Reply:
The regex captures both, but only the obejctID is being used.
The file type is retrieved from the file itself, instead of using the extension provided, the file

Reply:
If it is not being used, then simply match it, but why capture it?